cl/users/email_handlers.py

```diff
@@ -676,3 +677,101 @@ def enqueue_email(recipients: list[str], message_id: str) -> None:
             f"The message: {message_id} can't be enqueued because it "
             "doesn't exist anymore."
         )
+
+
+PERMANENT = {
```

Comment:
Following up on our conversation: The vast majority of these could be combined and I think don't provide much use to the end user.  If we can collapse these down into useful messages I think that would be smart.